README.md

```diff
@@ -3,7 +3,17 @@ Scripts and applications for testing DataBridges API from the API Gateway
 
 # Getting started
 ## How to install
-You can pip install the package
+You can use pip to install the package:
+```
+pip install databridgesapi@git+https://github.com/WFP-VAM/DataBridgesAPI
```

Comment:
Command update compared to #10 because of getting this error:

`Discarding git+https://github.com/WFP-VAM/DataBridgesAPI: Requested databridgesapi from git+https://github.com/WFP-VAM/DataBridgesAPI has inconsistent name: expected 'pip', but metadata has 'databridgesapi'`